### DISCUSSION TITLE
How to get all possible receivers for `-d {{receiver}}` of `termux-share` command?

### DISCUSSION THREAD

**Discussion by EmilyGraceSeville7cf** [termux/termux-app#4153] [Q&A]
Hi! How to get all possible receivers for `-d {{receiver}}` of `termux-share` command? I'm asking because I [have written](https://github.com/fish-shell/fish-shell/pull/10722) completions for Termux:API and wanna enhance them.

**Comment by agnostic-apollo** [comment]
The `-d` option is a flag and not an argument option. It does not expect anything. If you pass it, then the default app user has previously saved (`Always` button) for the intent content type will automatically be sent the intent directly, instead of android app chooser being shown.

https://github.com/termux/termux-api-package/blob/e62bdadea3f26b60430bb85248f300fee68ecdcc/scripts/termux-share.in#L35
